packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/OrderSummary.tsx

```diff
@@ -74,6 +100,40 @@ const OrderSummary: React.FC<Props> = ({
 
   const { days } = intervalToDuration({ end: lockTime, start: 0 })
 
+  const coinInterestRate = !!interestRate[outputCurrency] && interestRate[outputCurrency]
+
+  const isInterestEligibleCoin = useMemo(
+    () =>
+      !isEmpty(interestEligible) &&
+      orderState === 'FINISHED' &&
+      interestEligible[outputCurrency] &&
+      interestEligible[outputCurrency]?.eligible,
+    [interestEligible, outputCurrency, orderState]
+  )
+
+  const handleEarnRewardsButton = useCallback(() => {
+    analyticsActions.trackEvent({
```

Comment:
Not a huge deal, but I usually try to put tracking at the end of the handler so in case `analyticsActions.trackEvent` throws for some weird reason, the UX (`handleClose`, `setTimeout`) will still happen.

Reply:
I guess in this case it might not be caught by anything and would show an error to the user anyways. 🤷🏻

Reply:
sure, will change.